.github/workflows/main.yml

```diff
@@ -274,6 +276,8 @@ jobs:
           micromamba activate xeus-lite-host
           python -m pip install jupyterlite-xeus
           jupyter lite build --XeusAddon.prefix=${{ env.PREFIX }}
+          #cp xcpp.data _output/extensions/@jupyterlite/xeus/static
+          #cp $PREFIX/lib/libclangCppInterOp.so _output/extensions/@jupyterlite/xeus/static
```

Comment:
I don't think we run the `serve` command anywhere in the CI. We need these instructions just before serving. Hence commented out for now as I am not sure.